dlk/python/dlk/templates/src/matrix/multiplication.cpp

```diff
@@ -80,7 +80,7 @@ void matrix_multiplication_impl(
   for (std::size_t j = 0; j < B.cols(); j += regblock_m) {
     if (B.cols() - j >= regblock_m) {
       std::size_t k = 0;
-      for (; k < B.rows(); k += regblock_m) {
+      for (; k + regblock_m - 1 < B.rows(); k += regblock_m) {
```

Comment:
How about changing this condition to `k + regblock_m <= B.rows()`? It looks a little bit simple. (Though generated assembler code might be the same...)

Reply:
Fixed.